package.json

```diff
@@ -135,13 +136,12 @@
     "web3-utils": "^1.7.3"
   },
   "optionalDependencies": {
-    "fsevents": "^2.3.2"
+    "fsevents": "2.3.2"
```

Comment:
Why the pin?

Reply:
I think this is a remnant of me trying things out. Removed, I think with no consequences.